tests/test_s3_util_class.py

```diff
@@ -1,56 +1,35 @@
 from python_poetry_template.s3_util_class import S3Util
 
-from mock import Mock, patch
+from mock import MagicMock
 
 
-@patch("python_poetry_template.s3_util_class.boto3")
```

Comment:
No longer have to patch. Can create the mock directly and pass it in when instantiating the class.